src/net.cpp

```diff
@@ -2470,7 +2470,7 @@ void CNode::BeginMessage(const char* pszCommand) EXCLUSIVE_LOCK_FUNCTION(cs_vSen
     ENTER_CRITICAL_SECTION(cs_vSend);
     assert(ssSend.size() == 0);
     ssSend << CMessageHeader(Params().MessageStart(), pszCommand, 0);
-    LogPrint("net", "sending: %s ", SanitizeString(pszCommand));
+    LogPrint("net", "sending: %s\n", SanitizeString(pszCommand));
```

Comment:
Are you sure this is correct? I think this was supposed to be un-terminated, because other message-related commands continue the logging.